css/style.css

```
@@ -1,3 +1,62 @@
-h1 {
-    color: blue;
+body {
+    margin: 0;
+    font-family: Noto Sans JP;
+    background-color: #fff;
+    color: #333;
+}
+
+.site-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color: #61472E;
+    color: #fff;
+    padding: 10px 30px;
+}
+
+.logo {
+    width: 300px;
+    height: auto;
+    display: block;
+    margin: 0;
```

画像を `display: block; ` にするのはよく使うテクニックなのだけど、ここでは `vertical-align: bottom;` で対応するのがいいかも
```suggestion
    vertical-align: bottom;
```